client/src/pages/__snapshots__/email-sign-up.test.js.snap

```diff
@@ -162,7 +162,7 @@ exports[`<EmailSignUp /> Non-Authenticated user "not accepted terms and conditio
       />
       <a
         className="big-cta-btn btn btn-lg btn-primary btn-block"
-        href="http://localhost:3000/signin"
+        href="https://3000-leonfeng-freecodecamp-z4lp4gb2uwf.ws-us105.gitpod.io/signin"
```

Comment:
I don't believe this should be changed.

Reply:
Thanks for reviewing my PR, Naomi. I'm currently studying Gitpod documentation. Will @ mention you when I think everything is fixed.